test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -105,7 +111,7 @@ Describe "Test-Connection" -tags "CI" {
             $error[0].Exception.InnerException.ErrorCode | Should -Be $code
         }
 
-        It "Force IPv4 with implicit PingOptions" -Pending {
+        It "Force IPv4 with implicit PingOptions" {
```

Comment:
Please add comment as to why it is marked 'pending'.

Reply:
The pending was removed from this test?